api/src/config/deploy.js

```diff
@@ -1,14 +1,28 @@
+/**
+ * Exports configuration necessary to communicate with Scholly to deploy
+ * a client website to GitHub Pages.
+ */
 require('dotenv').config();
 
-const githubClientId = '03757c278dedad4b307d';
+/// Hostname of the scholly service.
+let schollyHost;
+/// Client ID of the configured GitHub OAuth application.
+let githubClientId;
+
+// Values depend on whether we're running in dev mode or prod (in containers).
+if (process.env.NODE_ENV === 'production') {
+  schollyHost = 'http://scholly:8000';
```

Comment:
Is this just referring to the container name like we saw in your demo video?

Reply:
Yep, `scholly` resolves to the IP of the container.

Reply:
Read https://docs.docker.com/compose/networking/

```
y default Compose sets up a single network for your app. Each container for a service joins the default network and is both reachable by other containers on that network, and discoverable by them at a **hostname identical to the container name.**
```